### PR TITLE
AppBar: Bottom bar with text - examples

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/769
[Problem] Bottom bar is not implemented
[Solution]
 - added bottom bar examples according to guideline

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/oneui-common.less [hunk 1]
```diff
@@ -26,6 +26,7 @@
     --appbar-main-text-color: @appbar-main-text-color;
     --appbar-subtitle-color: @appbar-sub-title-color;
     --appbar-miltiline-title-color: @appbar-multiline-title-color;
+    --bottom-bar-color: @bottom-bar-color;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/oneui-common.less:29]: How about using the @background-color directly without @bottom-bar-color?

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	<div class="ui-page" id="simpleTitleBack">
+		<header>
+			<div class="ui-appbar-left-icons-container">
+				<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+			</div>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">Sample title</span>
+			</div>
+			<div class="ui-appbar-action-buttons-container">
+					<button class="ui-btn ui-btn-icon ui-btn-icon-more" data-style="flat"></button>
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html]: Need to check indent

File: src/css/profile/mobile/common/oneui-common.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/oneui-common.less:29]: This color is different for light and dark color.

This is `in progress` PR, you don't have to do review 'in progress' PRs.

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html]: ok

File: src/css/profile/mobile/common/oneui-common.less (hunk 1 above)
Comment (HunseopJeong) [reply src/css/profile/mobile/common/oneui-common.less:29]: Sorry, I didn't watch the `in progress` I will review it later.

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/oneui-common.less:29]: No problem... I just want to say that this is changing and there is nothing to waste of energy. Anyway, Thank you for review.

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/oneui-common.less:29]: But, I have a problem with:
`@bottom-bar-color: @background-color;`
The less doesn't include proper color for dark theme.
When I set in this place:
`@bottom-bar-color: #f2f2f2`
Everything is ok.
It seems like less.js change reference to variable or get it from cache.

(update) The problem was solved by itself.

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-1-tab.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="simpleTitleBack">
```
Comment (grzegorz-czajkowski) [inline examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-1-tab.html]: Can we remove id="simpleTitleBack" ?

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-2-tabs.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="simpleTitleBack">
```
Comment (grzegorz-czajkowski) [inline examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-2-tabs.html]: Ditto.

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="simpleTitleBack">
```
Comment (grzegorz-czajkowski) [inline examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html]: Ditto.

**Comment by TomaszLukawskiSam** [comment]
Please check responsive UI also... widht-min: 685px

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-1-tab.html (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-1-tab.html]: Fixed

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-2-tabs.html (hunk 4 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-2-tabs.html]: Fixed

File: examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html (hunk 5 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/appbar/bottom-bar/bottom-bar-text-3-tabs.html]: Fixed

**Review by grzegorz-czajkowski — APPROVED** [review]
Works fine.

**Pull request merged**